docs/private-package-registry.md

```diff
@@ -0,0 +1,39 @@
+# Private package registry
+
+This guide describes how to build a Docker image with Cog that fetches Python packages from a private registry during setup.
+
+## `pip.conf`
+
+Create a `pip.conf` file with an `index-url` set to the registry's URL with embedded credentials.
```

Comment:
Great overview!

Question: Should we advise to either have a .dockerignore or (preferably?) keep this file outside of the cog project directory - as all content gets copied in your `cog build` step?

(Many times when I review customers' images they have lots of extra stuff that the they don't realize is copied in automatically.)

Reply:
Great idea. I just made a few additions that call this out explicitly.